frontend/src/components/mapbox.js

```diff
@@ -37,52 +42,64 @@ class MapBox extends React.Component {
       maxBounds: MAX_BOUNDS
     });
 
+    // TODO: set a default start date?
     getResponseRatesByDate("03312010").then(data => {
-        var responseRates = data.data.result.response_rates;
-        var tractData = responseRates.map(response_rate => {
-            return { GEOID: response_rate.tract_id, response_rate: response_rate.rate }
+      var responseRates = data.data.result.response_rates;
+
+      // We're going to map IDs to response rates so we can display them
+      var tractToRates = {};
+      responseRates.forEach(response_rate => {
+        tractToRates[response_rate.tract_id] = response_rate.rate;
+      });
+      this.setState({
+        tractToRates: tractToRates
+      });
```

Comment:
non-blocking, but do we need tractToRates in addition to tractData? is there a way we could use just one?

Reply:
you make a really good point lemme try something